File: Leetcode/1957 - Delete Characters to Make Fancy String/solution.cpp
string makeFancyString(string s) {
    string res = s.substr(0, 2);
    for (int i = 2; i < s.size(); i++) {
        if (!(s[i] == res[res.size() - 2] && s[i] == res[res.size() - 1])) {
            res += s[i];
        }
    }
    return res;
}

File: Leetcode/1957 - Delete Characters to Make Fancy String/solution.js
var makeFancyString = function(s) {
    const res = [s[0], s[1]];
    for (let i = 2; i < s.length; i++) {
        if (!(s[i] === res[res.length - 1] && s[i] === res[res.length - 2])) {
            res.push(s[i]);
        }
    }
    return res.join('');
};